### ISSUE TITLE
[Qt Style Sheet Highlighter] Dynamic property not accepted

### ISSUE THREAD

**Issue by std-any-emplace** [Quick-Turn-Studio/CLionSupportForQt#24]
This snippet is accepted by Qt5.1x, but Qt Style Sheet Highlighter does not accept the dynamic property even in quoted form.
```
QSlider::groove:horizontal[prop=true]
{
	border: 1px solid #999999;
}
```

**Comment by grabusr** [comment]
Hi,
Version 0.1.8 is ready to download. Please let's us know if it solves your problems.

**Comment by std-any-emplace** [comment]
Problem fixed in 0.1.8.
